liby2/src/genericfrontend.cc

```diff
@@ -657,42 +651,19 @@ main (int argc, char **argv)
 
 // FIXME the whole option parsing sucks **** !
 
-    // Check, if debugger should be enabled
+    // old backward compatibility stuff
```

Comment:
It is also mentioned in https://github.com/yast/yast.github.io/blob/master/doc/environment-variables.md